### ISSUE TITLE
(One#26772699) [Add an action]: Keyboard focus indicator is not visible on the search results in Add an action screen.

### ISSUE THREAD

**Issue by ek68794998** [Azure/LogicAppsUX#4295]
### **User Experience:** 

Keyboard dependent people will get impacted if keyboard focus does not appear on any interactive element present on the application. As a result, the end user will not be able to know the focused element and might miss the control.

### **Repro Steps:** 

1. Open URL [Manage your flows \| Power Automate](<https://make.powerautomate.com/environments/7f0defc6-9dbe-4f62-9893-52abf0e9b56f/flows>) and sign in with Valid credentials. 
2. Click on My Flows on left nav. 
3. Select a flow, click 'Edit' button and select it. 
4. Edit page will appear, navigate to 'New Step' button and select it. 
5. Choose an Action section will appear on the screen 
6. Search for mails and Navigate through search results 
7. Verify Keyboard Focus appearance for the search results. 

### **Actual Result:** 

Keyboard focus indicator is not visible on the search results in Add an action screen.

**Observation:** While navigating on the search results present in the Add an action screen with keyboard tab key controls, focus indicator is not visible and able to active controls with enter key.

### **Expected Result:** 

Keyboard focus indicator should be visible four sides, around the controls present in Add an action screen.

### **MAS Reference:** 

[MAS 2.4.7 -- Focus Visible](<https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7b5ED5CC1A-4764-40CD-8F53-C8107BA0D5A2%7d&file=MAS%202.4.7%20%E2%80%93%20Focus%20Visible%20.docx&action=default&mobileredirect=true>)

### **ARIA Authoring Ref Link/Fluent Patterns Guidance /HITS Guidance:** 

[https://www.w3.org/WAI/ARIA/apg/patterns/tabs/](<https://www.w3.org/WAI/ARIA/apg/patterns/tabs/>)

### **UIA/DOM details available:** 

N/A

### **Reference Links:** 

- [Accessibility Insights!](<https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2faccessibilityinsights.io%2f&data=04%7c01%7cv-vaisgupta%40microsoft.com%7c280d623b6a4d41db15ec08d9be06f891%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637749759434143108%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000&sdata=fLA4DL4v2zFP5bdYABQcw98V%2BInLjYKXpTYgtGnuhfM%3D&reserved=0>) - Identify accessibility bugs before check-in and make bug fixing faster and easier. 

### Are you using new designer or old designer 

New Designer (With action panels)

### Screenshots or Videos 

![MAS2 4 7 Need Visible Focus indicator when traversing through items of search results in Add an action screen](<https://github.com/Azure/LogicAppsUX/assets/1350074/6d6ab512-7bdb-4c70-b70f-191f2311d393>)

### Browser 

- **OS Version:** Windows 11 Version 23H2 (OS Build 22631.3007) 
- **Browser:** Edge Version 120.0.2210.144 (Official build) (64-bit) 
- **\#URL:**[Manage your flows \| Power Automate](<https://make.powerautomate.com/environments/7f0defc6-9dbe-4f62-9893-52abf0e9b56f/flows>) 

### Additional context 

[One#26772699](<https://dev.azure.com/msazure/One/_workitems/edit/26772699>)

**Comment by hartra344** [comment]
Already fixed here: #4060
